README.rst

```diff
@@ -1364,6 +1364,11 @@ Choose from the list of available rules:
     ones; defaults to ``['getrandmax' => 'mt_getrandmax', 'rand' =>
     'mt_rand', 'srand' => 'mt_srand']``
 
+* **return_assignment**
+
+  Non global, static or indirect referenced variables should not be
```

Comment:
"indirectly referenced"?

Reply:
:+1:
I update this when everything else in this PR is approved